content/php-client/resources/measure-families.md

```diff
@@ -0,0 +1,61 @@
+## Measure family
+
+### Get a measure family 
+
+```php
+$client = new \Akeneo\Pim\AkeneoPimClientBuilder('http://akeneo.com/')->buildAuthenticatedByPassword('client_id', 'secret', 'admin', 'admin');
+
+/*
+ * Returns an array like this:
+ * [
+ *      'code'     => 'casebox',
```

Comment:
But existing in measure families [casebox](https://github.com/akeneo/pim-community-dev/blob/master/src/Akeneo/Bundle/MeasureBundle/Resources/config/measure.yml#L470)